src/NServiceBus.RavenDB.Tests/SubscriptionStorage/When_receiving_duplicate_subscription_messages.cs

```diff
@@ -32,7 +32,27 @@ public class When_receiving_duplicate_subscription_messages : RavenDBPersistence
         }
     }
 
- 
+    [Test]
+    public async Task should_not_create_additional_db_rows_even_if_event_version_changed()
```

Comment:
Seems that this tests:
- when we have two documents saved for 2 different **event** versions saved by the same endpoint, only one subscriber is returned (i.e. no duplication)

Should we also cover:
- when we have two documents saved for 2 different **endpoint** versions, only one subscriber is returned

?